fuse/nodefs/fsops.go

```diff
@@ -70,6 +70,8 @@ func (c *FileSystemConnector) lookupMountUpdate(out *fuse.Attr, mount *fileSyste
 
 // internalLookup - execute a lookup without affecting NodeId reference counts
 func (c *FileSystemConnector) internalLookup(out *fuse.Attr, parent *Inode, name string, header *fuse.InHeader) (node *Inode, code fuse.Status) {
+	c.forgetMu.RLock()
```

Comment:
Oh but protecting `child := parent.GetChild(name)` ( fsops.go:72 ) with the lock should be enough. That way GetAttr can run outside the lock. Will check if it still passes.